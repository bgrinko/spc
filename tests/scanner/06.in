'1sfsdf23213   12'