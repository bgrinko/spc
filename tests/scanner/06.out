1
1	19	NONE		1sfsdf23213   12
